modules/op-installing-pipelines-operator-in-web-console.adoc

```diff
@@ -4,41 +4,53 @@
+
+If you have OpenShift Pipelines already installed on your cluster, the existing installation is seamlessly upgraded. The operator will replace the instance of `config.operator.tekton.dev` on your cluster with an instance of `tektonconfigs.operator.tekton.dev` and additional objects of the other CRDs as necessary.
+
+[WARNING]
+====
+If you manually changed your existing installation, such as, changing the target namespace in the `config.operator.tekton.dev` CRD instance by making changes to the `resource name - cluster` field, then the upgrade path is not smooth. In such cases, the recommended work flow is to uninstall your installation and reinstall the {pipelines-title} Operator. 
+====
 
 [discrete]
 .Procedure
 
 . In the *Administrator* perspective of the web console, navigate to *Operators* -> *OperatorHub*.
 
-. Use the *Filter by keyword* box to search for `{pipelines-title} Operator` in the catalog. Click the *OpenShift Pipelines Operator* tile.
-+
-[NOTE]
-====
-Ensure that you do not select the *Community* version of the *OpenShift Pipelines Operator*.
-====
-+
-//image::op-installed-tile.png[]
-//Remove the note below once the community operator is removed from OperatorHub
+. Use the *Filter by keyword* box to search for `{pipelines-title}` Operator in the catalog. Click the *{pipelines-title}* Operator tile.
 
-. Read the brief description about the Operator on the *{pipelines-title} Operator* page. Click *Install*.
-+
-//image::op-install-subscription.png[]
+. Read the brief description about the Operator on the *{pipelines-title}* Operator page. Click *Install*.
 
 . On the *Install Operator* page:
++
+.. Select *All namespaces on the cluster (default)* for the *Installation Mode*. This mode installs the Operator in the default `openshift-operators` namespace, which enables the Operator to watch and be made available to all namespaces in the cluster.
+
+.. Select *Automatic* for the *Approval Strategy*. This ensures that the future upgrades to the Operator are handled automatically by the Operator Lifecycle Manager (OLM). If you select the *Manual* approval strategy, OLM creates an update request. As a cluster administrator, you must then manually approve the OLM update request to update the Operator to the new version.
+
+.. Select an *Update Channel*.
 
-  .. Select *All namespaces on the cluster (default)* for the *Installation Mode*. This mode installs the Operator in the default `openshift-operators` namespace, which enables the Operator to watch and be made available to all namespaces in the cluster.
+*** The *ocp-<4.x>* channel enables installation of the latest stable release of the {pipelines-title} Operator.
```

Comment:
I think we have `stable` channel instead of *ocp-<4.x>*

/cc @khrm @piyush-garg @nikhil-thomas